README.md

```diff
@@ -666,6 +668,9 @@ Base service-under-test URL. Default value is `http://localhost`.
 #### httpTimeout
 Timeout for any requests to Selenium server. Default value is `90000` ms.
 
+#### urlTimeout
+Timeout for requests to Selenium server done during execution of the `url` command. Default value is `httpTimeout`.
```

Comment:
Давай поправим коммент как выше и допишем, что-то вроде:
`It may be useful when opening url takes a long time (for example a lot of logic is executed in middlewares) and you don't want to increase the timeout for other commands`